Assets/Scripts/Movement.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class Movement : MonoBehaviour
+{
+    private int speed = 20;
+    private int rotationSpeed = 100;
+    private float acceleration = 0.2f;
```

Comment:
It doesn't seem like the acceleration variable is used anywhere? seems like it could be removed